code/game/objects/items/kitchen.dm

```diff
@@ -50,7 +50,7 @@
 		icon_state = "fork"
 		forkload = null
 
-	else if(user.zone_selected == BODY_ZONE_PRECISE_EYES)
+	else if(user.is_zone_selected(BODY_ZONE_PRECISE_EYES, 30))
```

Comment:
what is this number. it is not clear. please named parameter or DEFINE, whichever one is correct